pages/api/metrics.ts

```diff
@@ -47,152 +47,152 @@ export default async function handler(
   }
 
   // 2. treasuries balances
```

Comment:
Maybe above this line a comment saying something to the effect of "The below are commented to avoid slow DB queries and don't have consumers"?

Reply:
Great idea, I'll do so.